adapters/persistence/adapter-maria/src/main/java/com/caregiver/domain/Content.java

```diff
@@ -0,0 +1,16 @@
+package com.caregiver.domain;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Content {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "content_id")
+    private long content_id;
```

Comment:
```suggestion
    private long id;
```

다른 엔티티의 id 값들과 달리 content_id 로 하신 특별한 이유가 있는지 궁금합니다.

Reply:
헉;; id 가 됐어야 하는데 ㅠㅠ 꼼꼼한 리뷰 감사쓰! 👍